2d/benchmarks/dambreak_Colagrossi/helpers.py

```diff
@@ -1,12 +1,14 @@
 def CreateFig():
     from tables import  openFile
-    archive = openFile('dambreak_Colagrossi0.h5','r')
+    archive = openFile('dambreak_Colagrossi_p.h5','r')
     import dambreak_Colagrossi
     import dambreak_Colagrossi_so
     import matplotlib.tri as mtri
     from matplotlib import pyplot as  plt
     import numpy as np
     domain = dambreak_Colagrossi.domain
+    domain.L = dambreak_Colagrossi.tank_dim
```

Comment:
Oops, there was an indentation problem in SpatialTools, and the bounding box would be generated only for 3D domains! It has been fixed here: https://github.com/erdc-cm/proteus/pull/532/commits/d8100c5332f8c3fce4122e570f7dfd7a2d5d34a6